<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1133</width>
    <height>644</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" stretch="0,1">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="ToolWidget" name="toolWgt" native="true"/>
   </item>
   <item>
    <widget class="QWidget" name="widget" native="true">
     <widget class="DrawSetWidget" name="setWgt" native="true">
      <property name="geometry">
       <rect>
        <x>11</x>
        <y>11</y>
        <width>278</width>
        <height>581</height>
       </rect>
      </property>
     </widget>
     <widget class="QWidget" name="grapWgt" native="true">
      <property name="geometry">
       <rect>
        <x>296</x>
        <y>11</y>
        <width>541</width>
        <height>585</height>
       </rect>
      </property>
     </widget>
     <widget class="PropertyWidget" name="propertyWgt" native="true">
      <property name="geometry">
       <rect>
        <x>844</x>
        <y>11</y>
        <width>278</width>
        <height>581</height>
       </rect>
      </property>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>DrawSetWidget</class>
   <extends>QWidget</extends>
   <header>drawsetwidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ToolWidget</class>
   <extends>QWidget</extends>
   <header>toolwidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>PropertyWidget</class>
   <extends>QWidget</extends>
   <header>propertywidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
